bite-size-notes/week-4-e-stochastic bellman equation.md

```diff
@@ -0,0 +1,167 @@
+
+$$
+    \pi = {\mu_0, \mu_1,...,\mu_{N-1}}
+$$
+
+where $\mu_k$ maps states $x_k$ into controls $u_k = \mu_k(x_k)$, and satisfies the control constraints, i,e is such that $\mu_k(x_k)\in U_k(x_k)$ for all $x_k \in S_k$. Such policies will be called __admissible__. Polices are more general objects than control sequences, and in the presence of stochastic uncertainty, they can result in improved cost, since they allow choices of controls $u_k$ that incorporate knowledge of the state $x_k$. Without this knowledge, the controller cannot adapt appropriately to unexpected values of the state, and as the result the cost can be adversely affected. This is a fundamental distinction between deterministic and stochastic optimal control problems.
+
+Another distinction between deterministic and stochastic problems is that the evaluation of various quatities such as cost function values involves forming expected values, and this may necessitate the use of Monte Carlo simulation. Monte Carlo simulation is a computational technique used to understand the impact of uncertainty and randomness in mathematical, engineering systems. In a Monte Carlo simulation, a model is run many times with random inputs, sampling from probability distributions that represent the uncertain parameters in the system. By repeating this process thousands or even millions of times, the simulation generates a range of possible outcomes and their probabilities.
+
+---
+## Stochastic Bellman value function Formation
+As following the same process of the deterministic value function we learned from the lecture, given an initial state $x_0$ and a policy $\pi = {\mu_0,..\mu_{N-1}}$, the future states $x_k$ and disturbances $\omega_k$ are random variables with probability distributions(i.e Guassian Normal Distribution) defined through the system equation,
+
+$$
+     x_{k+1} = f_k(x_k, \mu_k(x_k), \omega_k), \quad k=0,1,...,N-1
+$$
+
+Policies are more general objects than control sequences, and in the presence of stochastic uncertainty, they can result in improved cost, since they allow choices of controls $u_k$ that incorporate knowledge of the state $x_k$.
+
+Thus, for given functions $g_k$, $k=0,1,..., N$, the expected cost of the policy $\pi$ starting $x_k$ is :
+
+$$
+   V_{\pi}(x_k) = \mathbb{E} \[ g_N(x_N) + \sum^{N-1}_{m= k} g_k(x_m, \mu_k(x_m), \omega_m) \]                
+$$
+
+where the expected value operation  $\mathbb{E}\{\cdot\}$ is over all the random variables $\omega_k$ and $x_k$. An optimal policy $\pi^*$ is one that minimizes this cost; i.e,
+
+$$
+    V_{\pi^*}(x_k) = \min_{u_m \in U_m(x_m), m=k,..,N-1} \mathbb{E} \[ g_N(x_N) + \sum_{m=k}^{N-1} g_m(x_m, \mu_k(x_m), \omega_m) \] 
+$$
+

```

Comment:
I'm not quite get how the first equation evolve to the second line

Reply:
Oh, it simply decoupling the current state and the posterior state for the value operation. But the second line has the typo, so I'll change it.  Sorry for making confusion. 

Re: Add description for equation expansion and fixed the equation.